solidity/utils/multisig.js

```diff
@@ -0,0 +1,156 @@
+      resolve(data.toString().trim());
+    });
+  });
+};
+
+const getGasPrice = async (web3) => {
+  while (true) {
+    const nodeGasPrice = await web3.eth.getGasPrice();
+    const userGasPrice = await scan(`Enter gas-price or leave empty to use ${nodeGasPrice}: `);
+    if (/^\d+$/.test(userGasPrice)) {
+      return userGasPrice;
+    }
+    if (userGasPrice === "") {
+      return nodeGasPrice;
+    }
+    console.log("Illegal gas-price");
+  }
+};
+
+const getTransactionReceipt = async (web3) => {
+  while (true) {
+    const hash = await scan("Enter transaction-hash or leave empty to retry: ");
+    if (/^0x([0-9A-Fa-f]{64})$/.test(hash)) {
+      const receipt = await web3.eth.getTransactionReceipt(hash);
+      if (receipt) {
+        return receipt;
+      }
+      console.log("Invalid transaction-hash");
+    } else if (hash) {
+      console.log("Illegal transaction-hash");
+    } else {
+      return null;
+    }
+  }
+};
+
+const send = async (web3, account, gasPrice, transaction, value = 0) => {
+  while (true) {
+    try {
+      const gasEstimate = await transaction.estimateGas({ from: account.address, value: value });
+      console.log("gasEstimate: " + gasEstimate, " - value - ", value);
+      const tx = {
+        to: transaction._parent._address,
+        data: transaction.encodeABI(),
+        gas: Math.max(await transaction.estimateGas({ from: account.address, value: value }), MIN_GAS_LIMIT),
+        gasPrice: gasPrice || (await getGasPrice(web3)),
+        chainId: await web3.eth.net.getId(),
+        value: value,
+      };
+      const signed = await web3.eth.accounts.signTransaction(tx, account.privateKey);
+      const receipt = await web3.eth.sendSignedTransaction(signed.rawTransaction);
+      return receipt;
+    } catch (error) {
+      console.log(error.message);
+      const receipt = await getTransactionReceipt(web3);
+      if (receipt) {
+        return receipt;
+      }
+    }
+  }
+};
+
+const deployed = (web3, contractName, contractAddr) => {
+  const buildFile = JSON.parse(fs.readFileSync(path.join(ARTIFACTS_DIR, contractName + ".json"), { encoding: "utf8" }));
+  return new web3.eth.Contract(buildFile.abi, contractAddr);
+};
+
+let web3;
+let gasPrice;
+let account;
+
+const initialiseWeb3 = async () => {
+  web3 = new Web3(NODE_ADDRESS);
+  gasPrice = await getGasPrice(web3);
+  account = web3.eth.accounts.privateKeyToAccount(PRIVATE_KEY);
+  console.log("account: ", account);
+}
+
+const execute = async (transaction, ...args) => {
+  return await web3Func(send, transaction, ...args);
+};
+const web3Func = (func, ...args) => func(web3, account, gasPrice, ...args);
+
+const transferOwnership = async () => {
+  await initialiseWeb3();
+
+  const MULTISIG_ADDRESS = "0x924f5ad34698Fd20c90Fe5D5A8A0abd3b42dc711";
+
+  const converters = [
+    "0x52eD2aD69A44A431B393d28CE415c19348abAb16",
+    "0xf90558859d1bBC79544Cb2Da49bA7Ce471a6343e",
+    "0x303401D9AB4394AEaE4156F1A21C620eED56E22C"
+  ];
```

Comment:
please pass params instead of hardcoding

Reply:
Human Error while passing parameters vs Typos in hardcoding.

Latter can be solved with reviewers.

Agree, not the ideal method, but the necessary evil for now. Ideal would be something of fetching from a unified list of contracts like we have `mainnet.json` and `testnet.json` in Sovryn-smart-contract repo.